extension/src/paymentHandler/payment-handler.js

```diff
@@ -26,27 +26,28 @@ const PAYMENT_METHOD_TYPE_KLARNA_METHODS = [
 async function handlePayment(paymentObject, authToken) {
   if (!_isAdyenPayment(paymentObject))
     // if it's not adyen payment, ignore the payment
-    return { success: true, data: null }
-
+    return { success: true, actions: [] }
```

Comment:
Do we need success flag?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/656/commits/3706831713c202a1f0bc0671a40ecc4ffe52a098